src/__tests__/transmission_test.js

```diff
@@ -624,41 +628,31 @@ describe("base transmission", () => {
           res.writeHead(200, { "Content-Type": "application/json" });
           res.end("[{ status: 666 }]");
         },
-        // because this number is longer than our 5000 global test timeout, as well as (more importantly)
-        // the `timeout: 2000` below.
-        7500
+        // This number needs to be more than the `timeout: 2000` below,
+        // but less than the global test timeout of 5000 so that the server closes in time
+        // before jest starts complaining.
+        2500
```

Comment:
Would it be silly to set a constant for Transmission test timeout and use that constant here with 500 added? Make these numbers move together?

Reply:
I like that idea a lot, since I always find setTimeout confusing, with the number hanging around half a mile from the function name.